### DISCUSSION TITLE
对于划选文字自动就复制功能的一点点想法

### DISCUSSION THREAD

**Discussion by welovej** [shuax/MouseInc.Settings#216] [Ideas]
现在的是划选就自动复制, 按ctrl不复制, 这当然没有太大问题. 
但我想法是, 能不能反过来呢? 按ctrl+划选文字才是复制, 默认不是复制的, 这样感觉更顺滑, 谢谢作者~~

**Comment by shuax** [comment]
你说的也有道理，但是用着也有点麻烦，就不改了。

**Comment by welovej** [comment]
希望能改下吧，能主动触发进行复制比较克制，而且不容易因为无意识的划选造成自动复制的困扰，谢谢😜



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年1月13日(周四) 中午11:14
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [shuax/MouseInc.Settings] 对于划选文字自动就复制功能的一点点想法 (Discussion #216)




 
你说的也有道理，但是用着也有点麻烦，就不改了。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by cloudherder** [reply]
功能已经做了，想方便无非调整下设置和自己的习惯。太麻烦作者不好。我把选中自动复制关了，把向上画直线的手势设置成复制。这样连键盘都不用按。好像还更方便呢。

**Comment by welovej** [comment]
既是开了纳言之门，看作者心情吧



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年1月13日(周四) 中午11:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [shuax/MouseInc.Settings] 对于划选文字自动就复制功能的一点点想法 (Discussion #216)




 
功能已经做了，想方便无非调整下设置和自己的习惯。太麻烦作者不好。我把选中自动复制关了，把向上画直线的手势设置成复制。这样连键盘都不用按。好像还更方便呢。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by kevzcr** [reply]
这个建议非常好。支持。

**Comment by gbsix** [comment]
虽然可能不符合其他人的习惯，但是我觉得有道理。全局复制很容易覆盖掉剪贴板内容且无法恢复，我以前就是这样，所以把这个功能给关了。

**Reply by 6Tom** [reply]
帮你个忙，win10以后有win+v，试一下

**Reply by gbsix** [reply]
> 帮你个忙，win10以后有win+v，试一下

win7.哈哈哈
